src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -37,14 +37,24 @@ static ucs_config_field_t uct_cuda_copy_iface_config_table[] = {
      "Max number of cuda events. -1 is infinite",
      ucs_offsetof(uct_cuda_copy_iface_config_t, max_cuda_events), UCS_CONFIG_TYPE_UINT},
 
-    {"BW", "10000MBs",
-     "Effective memory bandwidth",
-     ucs_offsetof(uct_cuda_copy_iface_config_t, bandwidth), UCS_CONFIG_TYPE_BW},
+    /* TODO: 1. Add separate keys for shared and dedicated bandwidth
+             2. Remove the "other" key (use pref_loc for managed memory) */
+    {"BW", "10000MBs,h2d:8300MBs,d2h:11660MBs,d2d:320GBs",
+     "Effective memory bandwidth", 0,
+     UCS_CONFIG_TYPE_KEY_VALUE(UCS_CONFIG_TYPE_BW,
+         {"h2d", "host to device bandwidth",
+          ucs_offsetof(uct_cuda_copy_iface_config_t, bw.h2d)},
+         {"d2h", "device to host bandwidth",
+          ucs_offsetof(uct_cuda_copy_iface_config_t, bw.d2h)},
+         {"d2d", "device to device bandwidth",
+          ucs_offsetof(uct_cuda_copy_iface_config_t, bw.d2d)},
+         {"other", "any other memory types combinations bandwidth",
```

Comment:
no need for `other` key - you can set default value without any key (see https://github.com/openucx/ucx/blob/master/src/uct/cuda/gdr_copy/gdr_copy_iface.c#L27)